modules/zarr/src/lib/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { ZarrArray } from 'zarr';
+
+import { loadMultiscales, guessTileSize } from './utils';
+import { FetchFileStore } from './storage';
+import ZarrPixelSource from './pixel-source';
+import type { Labels } from '../types';
+
+/**
+ * Initializes multiscale Zarr pixel source.
+ * @param source Root url to multiscale Zarr store, or Zarr store.
+ * @param labels Labels for each array dimension.
+ */
+export async function load<S extends string[]>(source: string | ZarrArray['store'], labels: Labels<S>) {
```

Comment:
dimension `labels` are parsed from the OME metadata. We use labeled dimensions to unify indexing zarr/tiff sources, but something to consider is whether dimensions need labels at all.

Reply:
Ideally loaders.gl should work if just provided a URL (in either node or the browser). It would be great if a valid "store" could be substituted in the case where someone has a custom zarr-backend: https://github.com/manzt/reference-spec-reader/blob/f7b7487379add147763987496eb1a5917b43f2ef/src/store.js#L15

Reply:
> We use labeled dimensions to unify indexing zarr/tiff sources

Is this similar to [xarray](https://github.com/pydata/xarray) labeling each dimension? It seems like a good fit for Zarr? I thought Zarr dimensions were always labeled...

Reply:
Somewhat similar. You can think of Zarr as large numpy arrays (with the option to add additional metadata). Without any extra metadata, Zarr arrays are "unlabeled". Libraries like `xarray` may use Zarr as a storage backend by adding structured metadata (JSON) in `.zattrs`. This metadata is separate from `.zarray` which contains the _array_-specific metadata (chunk size, shape, compression, dtype, etc).